products/firefox_android/localize_android.md

```diff
@@ -1,22 +1,28 @@
 
 You can see the [detailed schedule for each release here](https://wiki.mozilla.org/RapidRelease/Calendar).
 
+"Merge day" is when localizations move to the next train - and the last day available to include localization updates to the next Beta version if you work on tools like Pootle.
+
 Note: please make sure that you've already localized mozilla.org before you start working on Firefox for Android or any other projects.
 
 ## The importance of regularly checking your dashboard
 
-Please keep in mind during your entire localization process to regularly check [your locale's dashboard](https://l10n.mozilla.org/shipping/dashboard): this is extremely important since if you break something, and your locale is shipping on multi-locale builds (so, the builds that actually ship on the Play Store), you will need to fix it as soon as possible since it breaks the build for everyone!
+### Checking for errors
+
+Please keep in mind during your entire localization process to regularly check [your locale's team page](https://l10n.mozilla.org/teams/). You can just choose your locale from the list there, and bookmark it to easily access it in the future. 
+
+Once you are on your team page, under the section "Application & Sign-offs", take a look at the "Translation Status" column. If you have any **errors**, they will show up in red there. The number of missing strings will also appear in that same column. You can click on either of these links to see what's going on behind these notifications.
+
+If you notice any errors, they need to be fixed very quickly. This is extremely important because if you break something while your locale is shipping on multi-locale builds (so, the builds that actually ship on the Play Store), you will need to fix it as soon as possible since it breaks the build for everyone!
 
-Checking your dashboard regularly should be something you should start doing from the very beginning when you start localizing a project, so that it becomes a habit. This helps reduce the chance of breaking the builds in the future, once your locale is actually shipping, and generally helps catch errors earlier on. It also gives you an overview of your locale's health and status.
+### Checking for warnings
 
-When you are in your dashboard, and you notice that there is a red tick in your locale's "Action" column, this means there is a critical error that you need to fix. Clicking on your locale's "Status" column will give you an indication on what needs to be fixed, and where to find the error.
+You should also be checking your teams [Firefox for Android dashboards](https://l10n.mozilla.org/shipping/dashboard?tree=fennec_aurora&tree=fennec_beta) for any possible **warnings**. They would appear under the "Warnings" column. If you see you have a warning, then click on the column "Status" column on that same line. It will explain where the warning is so you can go in and fix it.
```

Comment:
Thanks!
Concerning the readme: I didn't change it because looking at it now, I don't really see where that would fit.
It already links to the Android and Firefox docs in the 1st bullet point, and then lists out what they contain. I would need to probably reformat all this it want to add direct links to these docs.
Should I just do that?